File: framework/languages/java/e2e/DiscoveryRegistryHa/Client/src/main/java/systems/zlink/e2e/discoveryregistryha/client/scenarios/DrC2EmbeddedRegistryScenario.java
package systems.zlink.e2e.discoveryregistryha.client.scenarios;

import systems.zlink.e2e.discoveryregistryha.client.support.ClientContext;
import systems.zlink.e2e.discoveryregistryha.client.support.DiscoveryApiResult;

public final class DrC2EmbeddedRegistryScenario implements ClientScenario {
    @Override
    public DiscoveryApiResult run(ClientContext context) {
        return context.runStandard();
    }
}

File: framework/languages/java/e2e-kotlin/DiscoveryRegistryHa/Client/src/main/kotlin/systems/zlink/e2e/kotlin/discoveryregistryha/client/Scenarios/DrC2EmbeddedRegistryScenario.kt
package systems.zlink.e2e.kotlin.discoveryregistryha.client.Scenarios

import systems.zlink.e2e.kotlin.discoveryregistryha.client.Support.ClientScenarioContext

object DrC2EmbeddedRegistryScenario {
    fun run(context: ClientScenarioContext) {
        context.runChannelScenario()
    }
}
